Community/ideAlarm/automation/lib/python/configuration.py.example

```diff
@@ -38,7 +38,9 @@ idealarm_configuration = {
             'armAwayToggleSwitch': 'Toggle_Z1_Armed_Away',
             'armHomeToggleSwitch': 'Toggle_Z1_Armed_Home',
             'mainZone': True,
-            'canArmWithTrippedSensors': False
+            'canArmWithTrippedSensors': False,
+            'pinCodeEntry': 'Z1_PinCode',
+            'pinCode': 1111
```

Comment:
I suggest setting the default PIN to None.

Reply:
Agreed, I'll make the change